spec/mainadditions.html

```diff
@@ -45,19 +48,190 @@ <h1>Mathematical Operations</h1>
+          <p>If an implementation does not support a conversion described above or if political rules for time _t_ are not available within the implementation, the result must be *+0*<sub>𝔽</sub>.</p>
+          <emu-note>
+            <p>It is recommended that implementations use the time zone information of the IANA Time Zone Database <a href="https://www.iana.org/time-zones/">https://www.iana.org/time-zones/</a>.</p>
+            <p>1:30 AM on 5 November 2017 in America/New_York is repeated twice (fall backward), but it must be interpreted as 1:30 AM UTC-04 instead of 1:30 AM UTC-05. LocalTZA(TimeClip(MakeDate(MakeDay(2017, 10, 5), MakeTime(1, 30, 0, 0))), false) is <emu-eqn>-4 &times; msPerHour</emu-eqn>.</p>
+            <p>2:30 AM on 12 March 2017 in America/New_York does not exist, but it must be interpreted as 2:30 AM UTC-05 (equivalent to 3:30 AM UTC-04). LocalTZA(TimeClip(MakeDate(MakeDay(2017, 2, 12), MakeTime(2, 30, 0, 0))), false) is <emu-eqn>-5 &times; msPerHour</emu-eqn>.</p>
+            <p>Local time zone offset values may be positive <i>or</i> negative.</p>
+          </emu-note>
+        </emu-clause>
+      </del>
 
-    <ins class="block">
-      <emu-clause id="sec-date.prototype.totemporalinstant">
-        <h1>Date.prototype.toTemporalInstant ( )</h1>
-        <p>The following steps are performed:</p>
+      <emu-clause id="sec-localtime" type="abstract operation">
+        <h1>
+          LocalTime (
+            _t_: a time value,
+          ): a Number
+        </h1>
+        <dl class="header">
+          <dt>description</dt>
+          <dd>
+            It converts _t_ from UTC to local time.
+            <ins>The local political rules for standard time and daylight saving time in effect at _t_ should be used to determine the result in the way specified in this section.</ins>
+          </dd>
+        </dl>
         <emu-alg>
-          1. Let _t_ be ? thisTimeValue(*this* value).
-          1. Let _ns_ be ? NumberToBigInt(_t_) &times; 10<sup>6</sup>.
-          1. Return ! CreateTemporalInstant(_ns_).
+          1. <ins>Let _localTimeZone_ be ! CreateTemporalTimeZone(DefaultTimeZone()).</ins>
+          1. <ins>If _localTimeZone_.[[OffsetNanoseconds]] is not *undefined*, then</ins>
+            1. <ins>Let _offsetNs_ be _localTimeZone_.[[OffsetNanoseconds]].</ins>
+          1. <ins>Else,</ins>
+            1. <ins>Let _offsetNs_ be GetIANATimeZoneOffsetNanoseconds(ℤ(_t_) &times; 10<sup>6</sup>, _localTimeZone_.[[Identifier]]).</ins>
+          1. <ins>Let _offsetMs_ be RoundTowardsZero(_offsetNs_ / 10<sup>6</sup>).</ins>
+          1. Return _t_ + <del>LocalTZA(_t_, *true*)</del><ins>𝔽(_offsetMs_)</ins>.
```

Comment:
The `<del class="block">` breaks the structured headers unfortunately, but it does still look cleaner with `<ins class="block">`.

Reply:
In what way does it break? Things seem to work for me with ecmarkup v12.0.2.
```sh
$ sed -n '/sec-localtime/,/\/emu-clause/p' spec/mainadditions.html 
      <emu-clause id="sec-localtime" type="abstract operation">
        <h1>
          LocalTime (
            _t_: a time value,
          ): a Number
        </h1>
        <dl class="header">
          <dt>description</dt>
          <dd>
            It converts _t_ from UTC to local time.
            <ins>The local political rules for standard time and daylight saving time in effect at _t_ should be used to determine the result in the way specified in this section.</ins>
          </dd>
        </dl>
        <del class="block">
          <emu-alg>
            1. Return _t_ + LocalTZA(_t_, *true*).
          </emu-alg>
        </del>
        <ins class="block">
          <emu-alg>
            1. Let _localTimeZone_ be DefaultTimeZone().
            1. Let _offsetParseResult_ be Completion(ParseTimeZoneOffsetString(_localTimeZone_)).
            1. If _offsetParseResult_ is a normal completion, then
              1. Let _offsetNs_ be _offsetParseResult_.[[Value]].
            1. Else,
              1. Let _offsetNs_ be GetIANATimeZoneOffsetNanoseconds(ℤ(ℝ(_t_) &times; 10<sup>6</sup>), _localTimeZone_).
            1. Let _offsetMs_ be RoundTowardsZero(_offsetNs_ / 10<sup>6</sup>).
            1. Return _t_ + 𝔽(_offsetMs_).
          </emu-alg>
        </ins>
        <p><ins>
          If political rules for the local time _t_ are not available within the implementation, the result is equal to _t_ because DefaultTimeZone returns *"UTC"* and GetIANATimeZoneOffsetNanoseconds returns 0.
        </ins></p>
        <emu-note>
          <p>Two different input time values <emu-eqn>_t_<sub>UTC</sub></emu-eqn> are converted to the same local time <emu-eqn>t<sub>local</sub></emu-eqn> at a negative time zone transition when there are repeated times (e.g. the daylight saving time ends or the time zone adjustment is decreased.).</p>
          <p><emu-eqn>LocalTime(UTC(_t_<sub>local</sub>))</emu-eqn> is not necessarily always equal to <emu-eqn>_t_<sub>local</sub></emu-eqn>. Correspondingly, <emu-eqn>UTC(LocalTime(_t_<sub>UTC</sub>))</emu-eqn> is not necessarily always equal to <emu-eqn>_t_<sub>UTC</sub></emu-eqn>.</p>
        </emu-note>
      </emu-clause>
$ node_modules/.bin/ecmarkup --version
ecmarkup v12.0.2
$ npm run build:spec && echo SUCCESS

> proposal-temporal@1.0.0 prebuild:spec
> mkdirp out


> proposal-temporal@1.0.0 build:spec
> ecmarkup --lint-spec --strict --load-biblio @tc39/ecma262-biblio spec.html out/index.html

SUCCESS
```
![screen](https://user-images.githubusercontent.com/1199584/167047097-2d367334-d4e8-4feb-bc25-cc75eea2b020.png)

Reply:
You'll notice the "It performs the following steps when called:" is missing if the first element after `</dl>` isn't `<emu-alg>`. (A minor breakage, for sure.)

Reply:
Oh, I think we can live that bug for now. Also, cc @bakkot. :point_up:

Reply:
https://github.com/tc39/ecmarkup/issues/443